libs/core/src/lib/tile/directives/numeric-content.directives.ts

```diff
@@ -0,0 +1,280 @@
+import { Directive, ElementRef, HostBinding, Input, OnChanges, OnInit } from '@angular/core';
+import { applyCssClass } from '../../utils/decorators/apply-css-class.decorator';
+
+type NumericContentState = 'negative' | 'critical' | 'positive' | 'informative' | '' | 'neutral';
+type NumericContentSize = 's' | 'm' | 'l' | '';
```

Comment:
Is there a need to add empty `''` strings into types? 
In case of no value I would suggest to keep null value, instead of empty string